velox/functions/prestosql/ArrayFunctions.h

```diff
@@ -418,4 +418,52 @@ struct ArrayFrequencyFunction {
   }
 };
 
+template <typename TExecParams, typename T>
+struct ArrayNormalizeFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(TExecParams);
+
+  FOLLY_ALWAYS_INLINE bool callNullFree(
```

Comment:
Does this function always return 'true'? If that's the case, it is more efficient to change return type to 'void'.

Reply:
Updated, thanks.